packages/components/organisms/f-footer/test/specs/component/f-footer.component.spec.js

```diff
@@ -60,11 +60,23 @@ describe('f-footer component tests', () => {
 
         // Act
         browser.back();
-        browser.switchToFrame(0);
         FooterComponent.waitForFooter();
         FooterComponent.clickYoutubeIcon();
 
         // Assert
         expect(browser.getUrl()).toContain('https://www.youtube.com');
     });
+
+    it('Does not show courier links on en-GB locale if courier links is set to false', () => {
+        // Assert
+        expect(FooterComponent.isCourierLinksDisplayed()).toBe(false);
+    });
+
+    it('Does not show courier links on en-GB locale if courier links is set to true', () => {
```

Comment:
Is this test correct? Looks like it should be asserting to `true`?

Reply:
Yup. This is correct. This is the GB test and we don't have any courier information to show just yet so this is supposed to assert `false`.